package.json

```diff
@@ -88,6 +86,7 @@
     "compression": "1.7.x",
     "credit-card-type": "6.3.x",
     "csrf": "3.1.x",
+    "eslint-plugin-import": "^2.26.0",
```

Comment:
Can this be a dev dependency ?

Reply:
Good spot - have updated this.
Just fixing conflicts with package-lock.json